10-Merge/readme.md

```diff
@@ -0,0 +1,202 @@
+git-merge
+=========
+
+> Join two or more development histories together
+
+We will start with warning from documentation:
+
+> Warning: Running git merge with non-trivial uncommitted changes is discouraged: while possible, it may leave you in a state that is hard to back out of in the case of a conflict.
+
+Let's all start with cloning testing repository to check merge on:
+
+	$ git clone git@github.com:DevTrainings/test_merge_conflict.git
+	$ git checkout bar && git checkout cherry-pick && git checkout foo && git checkout master
```

Comment:
* Please add cd test_merge_conflict
* Please split commands to individual lines - this does not work in powershell